lib/integrations/externalauditstorage/configurator.go

```diff
@@ -118,6 +127,27 @@ func (o *Options) setDefaults(ctx context.Context, region string) error {
 	return nil
 }
 
+func awsLoggerFunc(fields logrus.Fields) func(classification logging.Classification, format string, v ...interface{}) {
```

Comment:
you might have to also cast the func before returning it, but i think this makes the function signature more clear and eliminates the type case where it's called
```suggestion
func awsLoggerFunc(fields logrus.Fields) logging.LoggerFunc {
```